tests/unit/test_ods.py

```diff
@@ -0,0 +1,214 @@
+import mock
+import unittest
+import functools
+
+import app.helpers.ods as ods
+
+from hypothesis import strategies as st
+from hypothesis import given, example, assume
+
+po = functools.partial(mock.patch.object, autospec=True)
```

Comment:
I say give it a better name and put it in helpers. Repeating this in every test method doesn't add much. And pyflakes will tell you if any of the local variables clash with your imported name.